Automattic/wp-calypso
# Backup: migrate requestActivity data getter to React Query

Migrates the `requestActivity` data getter, used when displaying a site backup, to React Query.

**How to test:**
On an Atomic or Jetpack site, go to `/backup/:site`. You're likely to have at least one backup of your site. Click on "restore" or "download" and you'll see this page (example is for restore):

<img width="730" alt="Screenshot 2022-01-20 at 14 19 32" src="https://user-images.githubusercontent.com/664258/150347891-68bbe4e0-1cfc-45d5-9e8c-05b1eae704cb.png">

The route of this page is `/backup/:site/restore/:rewind_id`. Verify that the page loads and displays ok.

Then try to change the `:rewind_id` parameter to something else, an invalid ID, and verify that the failure to load is handled correctly:

<img width="731" alt="Screenshot 2022-01-20 at 14 20 06" src="https://user-images.githubusercontent.com/664258/150348081-68c6aa97-f0c2-4371-87f9-80bc9d6c6bdb.png">
